spec/models/claims/state_machine_scopes_spec.rb

```diff
@@ -40,4 +40,36 @@
       end
     end
   end
+
+  describe 'scoped associations' do
+    let(:claim) { create :advocate_claim }
+
+    describe '.archived_claim_state_transitions' do
+      it 'only returns transitions to archived state' do
```

Comment:
as for the other PR would be good to have make use of contexts and have test setup separate from the actual test.